packages/block-library/src/list/edit.js

```diff
@@ -128,7 +129,7 @@ export default function Edit( { attributes, setAttributes, clientId, style } ) {
 		...( Platform.isNative && { style } ),
 	} );
 	const innerBlocksProps = useInnerBlocksProps( blockProps, {
-		allowedBlocks: [ 'core/list-item' ],
+		allowedBlocks: ALLOWED_BLOCKS,
```

Comment:
The whole object passed as the second arg of this `useInnerBlocksProps()` call could be a constant, couldn't it?

Reply:
It could be, but I think we shallow compare it anyway, so there would be no point.